src/app/constants.ts

```diff
@@ -36,7 +36,9 @@ export const HAS_LETTER_OR_SYMBOL = /(?=.*[^\d\s]).*/
 // https://regex101.com/r/gznzc6/1 strips params for OJS links
 export const REDIRECT_URI_REGEXP = /(?=redirect_uri=)(.*?)(?=orcidapi)|(?=redirect_uri=)(.*?)$/
 // https://regex101.com/r/EP7uWn/1
-export const AMOUNT_REGEXP = /^[0-9.,]*$/
+export const AMOUNT_FORMATTED_WITH_DECIMAL_REGEXP = /^(\d+(\.\d{1,2})?|\.?\d{1,2})$/
```

Comment:
Camelia we normally add these comments of top of each REGEXP

 // https://regex101.com/r/....

 Each with a list of examples to make it easier to understand how the regexp works